modules/ingest-geoip/build.gradle

```diff
@@ -39,11 +39,12 @@ opensearchplugin {
 }
 
 dependencies {
-  api('com.maxmind.geoip2:geoip2:4.2.1')
+  api('com.maxmind.geoip2:geoip2:4.3.0')
   // geoip2 dependencies:
   api('com.maxmind.db:maxmind-db:3.1.1')
   api("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
   api("com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}")
+  api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${versions.jackson}")
```

Comment:
I had to add this manually. It was added in https://github.com/maxmind/GeoIP2-java/commit/308f098dec39071cac4e3a901f2523f982286cb3#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8 but failing to resolve transitively.